src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -59,22 +59,36 @@ bool IWorkspaceProjectContext.LastDesignTimeBuildSucceeded
                 _lastDesignTimeBuildSucceeded = value;
             }
         }
+
+        string IWorkspaceProjectContext.BinOutputPath
+        {
+            get
+            {
+                return this.TryGetBinOutputPath();
+            }
+            set
+            {
+                SetOutputPathAndRelatedData(objOutputPath: this.TryGetObjOutputPath(), binOutputPath: value);
```

Comment:
I have replaced the TryGetXXXOutputPath methods with XXXOutputPath properties in a separate PR: https://github.com/dotnet/roslyn/pull/12927/files#diff-c1adde957bdde82f66d830cea57b2ddfR165.